pandas/core/indexes/base.py

```diff
@@ -3867,8 +3867,8 @@ def _values(self) -> Union[ExtensionArray, ABCIndexClass, np.ndarray]:
 
         See Also
         --------
-        values
-        _ndarray_values
+        values : Return a Numpy representation of the DataFrame.
```

Comment:
"Return a Numpy array or Pandas  **ExtensionArray** of the Index" or similar

Reply:
This still applies. Index.values does not necessarily return a numpy array, but can also return other types.